File: frontend/my-app/src/lib/descripcion-producto.ts
/** Combina descripción y descripción extra separadas por guión. */
export function formatDescripcionProducto(
  descripcion?: string | null,
  descripcionExtra?: string | null
): string {
  const base = (descripcion ?? "").trim()
  const extra = (descripcionExtra ?? "").trim()
  if (extra) {
    return base ? `${base} - ${extra}` : extra
  }
  return base
}

/** Clase opcional para textos largos en listas compactas. */
export function descripcionProductoTextClass(text: string): string {
  return text.length > 55 ? "small" : ""
}

File: backend/src/descripcion_producto.py
from typing import Optional


def format_descripcion_producto(
    descripcion: Optional[str],
    descripcion_extra: Optional[str] = None,
) -> str:
    """Combina descripción y descripción extra separadas por guión."""
    base = (descripcion or "").strip()
    extra = (descripcion_extra or "").strip()
    if extra:
        return f"{base} - {extra}" if base else extra
    return base
